sippy-ng/README.md

```
@@ -1,59 +1,64 @@
 
 ```
-process.env.REACT_APP_API_URL = 'https://sippy.dptools.openshift.org'
+process.env.VITE_API_URL = 'https://sippy.dptools.openshift.org'
 ```
+
 ## Formatting requirements
 
-Formatting requirements are enforced on the order of imports (alphabetically) and [prettier](https://prettier.io/docs/en/options.html).  The [prettier config](prettier.config.js) can be modified to change the formatting standards.  From the command line prettier formatting can be applied via
+Formatting requirements are enforced on the order of imports (alphabetically) and [prettier](https://prettier.io/docs/en/options.html). The [prettier config](prettier.config.js) can be modified to change the formatting standards. From the command line prettier formatting can be applied via
+
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)